tests/tiup-cluster/script/scale_tools.sh

```diff
@@ -70,6 +74,14 @@ function scale_tools() {
     topo=./topo/full_scale_in_grafana.yaml
     tiup-cluster $client --yes scale-out $name $topo
 
+    echo "start scale out prometheus"
+    topo=./topo/full_scale_in_prometheus.yaml
+    wait_instance_num_reach $name $total_add_one $native_ssh
```

Comment:
This line should be moved below the scale-out operation

Reply:
Sorry，fix it ASAP.